pkg/agent/monitortool/monitor.go

```diff
@@ -128,7 +137,7 @@ func (m *NodeLatencyMonitor) onNodeAdd(obj interface{}) {
 
 	m.latencyStore.addNode(node)
 
-	klog.V(4).InfoS("Node added", "Node", klog.KObj(node))
+	klog.InfoS("Node added", "Node", klog.KObj(node))
```

Comment:
please revert all these logging changes. It may be better for you to increase verbosity at runtime when you test, rather than modify the code.

Reply:
Ok, I'll keep an eye on them.

Reply:
No addressed. I still see a lot of info logs for which the `V(4)` has been removed.

Reply:
I  recommited them, now it is fixed.